diag_manager/fms_diag_reduction_methods.F90

```diff
@@ -0,0 +1,102 @@
+!> \author Ganga Purja Pun
```

Comment:
If you and @rem1776 are writing this, they you should be listed as authors.  Especially because if this is `git blame`d, Ganga won't come up at all.

Reply:
I removed the author lists (that is what git is for), added the license that is all the other files, and doxygenized the module documentation